src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -710,6 +708,9 @@ bool QgsPostgresConn::getTableInfo( bool searchGeometryColumnsOnly, bool searchP
     if ( !schema.isEmpty() )
       sql += QStringLiteral( " AND %1='%2'" ).arg( schemaName, schema );
 
+    if ( !name.isEmpty() )
+      sql += QStringLiteral( " AND %1='%2'" ).arg( tableName, name );
```

Comment:
there's a risk of SQL injection here. QgsPostgresConn::quotedValue() shoud likely be used . Same for the schema at line 709 above.
And same for lines 852,855,994,997

Reply:
I've created https://github.com/qgis/QGIS/pull/56173 that should fix all the issues with current master, and be backportable to 3.34. I'm not sure how we want to handle the conflicts with that PR

Reply:
@rouault can we merge this one now? Then I will apply the quoting after yours is merged (or you can rebase and apply the quoting).

Reply:
fine for me